tests/consensus_tests/test_partial_snapshot.py

```
@@ -198,8 +216,8 @@ def recover_collection_snapshot(peer_url: str, snapshot_url: str):
     return resp.json()["result"]
 
 def recover_partial_snapshot_from(peer_url: str, recover_peer_url: str, wait = True):
-    resp = try_recover_partial_snapshot_from(peer_url, recover_peer_url)
-    assert_http_ok(resp)
+    resp = try_recover_partial_snapshot_from(peer_url, recover_peer_url, wait)
+    assert_http_ok(resp) # this should be `assert res.ok`
```

Ruff (0.12.2)

`assert_http_ok` may be undefined, or defined from star imports

(F405)